Ejercicio11 seba

giunta opened the pull request:
el ejercicio 11 ya con el main y el header editados; 
lo verifique con varios positivos y me andaba bien y con los negativos creo que tambien pero no se muy bien como verificarlos
y tambien no tenia idea de que tenia que escribir esto aca al final jajaja creo que lo escribi unas 3 veces

giunta committed:
ejercicio 11

el ejercicio 11 con el main y el header ya esditado

giunta committed:
Update ejercicios.h

giunta committed:
ej11.c

el ejercicio 11,  probe con varios positivos y me anduvo bien con todos; 
los negativos no se muy bien como comprobar si estan bien pero me parece que anda

FernandezGFG commented:
Joyitaaaa 👏 👏  Ahora lo miro

GinoAvanzini committed:
Update README.md

GinoAvanzini committed:
Add files via upload

Ejercicio 9

GinoAvanzini committed:
Update ejercicios.h

GinoAvanzini committed:
Update main.c

GinoAvanzini commented:
Agregué a tu branch lo del ejercicio 9 para no hacer dos pull request

GinoAvanzini committed:
Update main.c

GinoAvanzini committed:
Update ej9.c

Itereichons

ichmew committed:
Add files via upload

Ejercicio 10, no sé si alguien lo había subido ya

ichmew committed:
Update main.c

ichmew committed:
ejercicio10

No le den bola al archivo ej10alt. Este ej10 es el posta.

ichmew committed:
ejercicio 12

Ej12:
Revisen el tamaño de los vectores que usé, capaz que le puedan encontrar una mejor solución. Ahí en el mismo archivo puse mi razonamiento.

giunta commented:
justo estaba haciendo el 12 jaja ahora me pongo con el 13 entonces

ichmew committed:
Ejercicio 13

Soy un cldo Seba no vi tu mensaje jajaja bueno ahora hago el 15

giunta commented:
mierda ya lo hice el 13 tambien lo voy a subir en otra branch asi lo comparan con el tuyo

ichmew commented:
Dale buenísimo, cuando puedas pegale una mirada al 12.

giunta committed:
Update main.c

giunta committed:
Update ejercicios.h

giunta committed:
Create ej17.c

giunta commented:
ahi acabo de subir el ejercicio 17

giunta commented:
el ejercicio 12 tuyo esta muy bueno andres

ichmew commented:
Puta que boludo subí el ej15 al master.
Cómo lo bajo de ahí compañeros fanes expertos en Github??

giunta commented:
yo digo que ya fue
lo podemos editar desde ahi si tiene algo mal

ichmew committed:
Update main.c

Ahí agregamos con el Seba la pregunta Y/N y quedó joya!!

giunta committed:
Update main.c

giunta committed:
Update ejercicios.h

giunta committed:
ejercicio 14

ahí subo el ejercicio 14; lo probé de varias formas y esta andando joya
también lo edite el main y los headers

ichmew committed:
Ejercicio 16

Falta pulir la impresión en pantalla pero anda joya. Necesitaría saber si hay alguna manera de que a los valores de float los muestre con menos decimales.

ichmew committed:
Update main.c

ichmew committed:
Update ejercicios.h

ichmew committed:
Update ej16.c

Solucionado el problema de la impresión en pantalla. Ahora me doy cuenta que no me tiraba los decimales en el float, directamente truncaba el valor y le agregaba .0000 jaja. Si alguién quiere pegarle una mirada a eso bienvenido sea XD.

giunta committed:
Update ejercicios.h

giunta committed:
Update main.c

GinoAvanzini committed:
Update ejercicios.h

GinoAvanzini committed:
Delete ej10alt.c

Gonza gil

GinoAvanzini committed:
Update main.c

GinoAvanzini committed:
Update ejercicios.h

gonzafernan committed:
Merge branch 'master' into ejercicio11-seba
